src/components/add-form/index.js

```diff
@@ -18,38 +23,63 @@ const AddForm = ({ formState, handleForm }) => (
         required
       />
     </label>
-    <fieldset className="time-frame" defaultValue="7">
+    <fieldset className="time-frame">
       <legend className="time-frame-legend">
         How soon will you buy this again?
       </legend>
-      <RadioInput
-        onChange={(event) => handleForm(event)}
-        value="7"
-        name="timeFrame"
-        label="Soon"
-        aria-label="timeFrame Soon"
+
+      <label
+        htmlFor="soon"
+        aria-label="You need to buy this product soon"
         className="time-frame-radio"
-        required
-        checked
-      />
-      <RadioInput
-        onChange={(event) => handleForm(event)}
-        value="14"
-        name="timeFrame"
-        aria-label="timeFrame Kind of Soon"
-        label="Kind of Soon"
+      >
+        <input
+          required
+          type="radio"
+          id="soon"
+          name="timeFrame"
+          value={TimeFrames.soon}
+          onChange={(event) => handleForm(event)}
+          checked={formState.timeFrame === TimeFrames.soon}
+        />
+        Soon
+      </label>
+      <br />
```

Comment:
Note that in this context, `<br />` is not semantic. I'd suggest using another approach to add a space between input elements.

One way could be adding a [gap](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) value in the parent element, so each `label` can have its own padding (or gap). Up to you if you thing of a better approach for this :).

e.g.

```css
fieldset {
  ...
  gap: 16px;
}
```

According to W3C [^1] and MDN [^2], `<br />` should be used as line breaks inside content (e.g. inside `<p>something <br /> new line</p>`).

[^1]: https://www.w3.org/TR/2014/REC-html5-20141028/text-level-semantics.html#the-br-element.
[^2]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
Hi @jandrade! Thank you very much for the suggestion! I didn't know this property, it will definitely help a lot! However, while making corrections I removed the tag and the styles remained the same. The `<br />` were unnecessary, apparently 😄